README.md

```diff
@@ -2645,6 +2646,8 @@ security:
     scopes: ["openid"]
     # You may optionally specify a list of allowed subjects. If this is not specified, all subjects will be allowed.
     #allowed-subjects: ["johndoe@example.com"]
+    # You can specify a different claim field to match against allowed-subjects. If not specified "subject" is used.
```

Comment:
This isn't accurate. The field name is `sub`, not `subject`. This also kind of implies that the value of the claim-to-check parameter will be set to that value, which isn't correct, as that may in fact not work (not sure, but I think the claims map is just the custom claims, not the default fielde like `sub`)